### ISSUE TITLE
Ultra Setting Lag

### ISSUE THREAD

**Issue by RedPrideGames** [dalaranwow/dalaran-wow#5274]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

Hello guys
new member and I have already come across a major issue
I can play Retail WoW on ultra settings with loads of Addons on and it works smoothly.
I've been playing this Private Server for a couple days now and all of a sudden out of the blue today (1/27/2018) it is very laggy

I have to play on Good settings to avoid the huge lag spikes and I had to disable some useful addons.

Please fix this issue, the beautiful thing about wotlk was enjoying the beautiful scenes while questing before Deathwing destroyed everything, its one of the reasons I hopped on this Private Server. 

I would provide screenshots but you cant screen shot a lag....

**Comment by Golganeth** [comment]
Not a server-side issue. Please check out your WoW client, addons, drivers, and anything related to your computer that could be causing this.

Closing up.

**Comment by RedPrideGames** [comment]
Already have...its not my computer

**Comment by Golganeth** [comment]
It is not a server-side issue so it must be on your side, since other players can play with those settings while having 0 issues. Check everything, from your computer to your client. 

Cheers
